server/server.cpp

```diff
@@ -13250,11 +13249,11 @@ int main() {
 
                                     // Get the substr from one after the seed delim
                                     std::string seed { emailAndSeed.substr( seedDelimPos + 1 ) };
-                                    char *sSeed = SettingsManager::getStringSetting("seedSalt", "default salt");
-                                    std::string seedSalt { sSeed };
+                                    char *sSeed = SettingsManager::getStringSetting("seedPepper", "default pepper");
```

Comment:
is there a reason sSeed is created if it's just used to initialize a string and deleted afterwards?